build/js/intlTelInput.js

```diff
@@ -636,11 +644,27 @@ https://github.com/Bluefieldscom/intl-tel-input.git
         },
         // decide where to position dropdown (depends on position within viewport, and scroll)
         _setDropdownPosition: function() {
-            var inputTop = this.telInput.offset().top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
-            dropdownFitsBelow = inputTop + this.telInput.outerHeight() + this.dropdownHeight < windowTop + $(window).height(), dropdownFitsAbove = inputTop - this.dropdownHeight > windowTop;
+            var showDropdownContainer = this.options.dropdownContainer && !this.isMobile;
+            if (showDropdownContainer) this.dropdown.appendTo(this.options.dropdownContainer);
+            var that = this, pos = this.telInput.offset(), // show the menu and grab the dropdown height
+            dropdownHeight = this.countryList.removeClass("hide").outerHeight(), inputTop = pos.top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
```

Comment:
With this method, it's not necessary to add/remove the `v-hide` class. I moved `this.countryList.removeClass("hide");` from the `_showDropdown` function into the `_setDropdownPosition` function. This allows us to calculate dropdownHeight before doing anything else in `_setDropdownPosition` (but after the dropdownContainer has been appended). So, yes, the menu will be visible, but that's what we want, as it allows us to get this.countryList's outerHeight.

Basically, instead of adding `v-hide` and removing `hide`, then calculating the height, and then removing `v-hide` and adding `hide`, and then later removing `hide` when the menu is open, we're just calculating the height when `hide` is removed, as we have to do that anyway.

I thought about caching dropdownHeight after it's calculated (so it doesn't need to be recalculated on subsequent openings), but we have to do `removeClass("hide")` anyway, and doing `outerHeight()` isn't really a performance hit. This also leaves open the option in the future to allow for more dynamic menu heights - cases where it might be desirable to have the menu height change between openings (like if countries have been removed from or added to the list).